projects/objectnav_baselines/experiments/objectnav_mixin_rawgru.py

```diff
@@ -0,0 +1,59 @@
+from typing import Sequence, Union
```

Comment:
`raw` doesn't quite evoke the meaning I think we want here. How about `objectnav_mixin_finetunecnn_gru.py ` or `objectnav_mixin_unfrozenresnet_gru.py ` or similar?